web-setup.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/sh
+
+# Author : Anthony Bartman
+# Description : This will install all neccessary files for
+# our Flask backend server. Mostly from this tutorial: 
+# https://www.codementor.io/@dongido/how-to-build-restful-apis-with-python-and-flask-12qto530jd
+
+echo ""
+echo ""
+echo "Running Flask Backend Install script."
+echo "This may take a few moments..."
+echo ""
+
+#Use Python 3.7 on board, not using Python2
+sudo update-alternatives --install /usr/bin/python python /usr/bin/python3 7
+
+#Install Pip (If Necessary)
+sudo apt update
+sudo apt install python3-pip
+pip3 --version
+echo " - Installed pip3"
+
+#Install Virtual Env. to create isolated Python Env.
+#WE DO NOT NEED A VIRTUAL ENVIRONMENT FOR OUR SENIOR DESIGN PROGRESS
+# pip3 install virtualenv
+# virtualenv --version
+# echo " - Installed virtualenv"
+# Create Virtual Environment
+# python -m venv env
+# echo " - Virtual Environment Created"
```

Comment:
Why don't we need a virtual environment? It would be helpful to make sure we all have the same python and packages running to avoid the "it works on my machine"

Reply:
I am pretty sure we will not need a virtual environment because that is really only for let's say having multiple python projects at once so that each python project will have its own individual dependencies.  But my thought for our project is that we will only be using one environment for python anyway and won't be needing a virtual one.